config/settings/_base.py

```diff
@@ -60,14 +60,15 @@
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.humanize',
+    'django.contrib.messages',
     'django.contrib.sessions',
     'django.contrib.sites',
-    'django.contrib.messages',
-    'whitenoise.runserver_nostatic',
+    'django.contrib.sitemaps',
     'django.contrib.staticfiles',
-    'raven.contrib.django.raven_compat',
 
     'django_celery_beat',
```

Comment:
Re-org'd to be:

- django libs
- 3rd party libs
- project libs.

Not quite relevant but I find it more intuitive.

Reply:
I would normally do this, but `whitenoise` needs to be before `runserver_nostatic` (at least it did, if not anymore please point me to the changelist).